x/mint/module.go

```diff
@@ -11,6 +11,7 @@ import (
 	"cosmossdk.io/core/appmodule"
 	"cosmossdk.io/core/registry"
 	"cosmossdk.io/x/mint/keeper"
+	mintKeeper "cosmossdk.io/x/mint/keeper"
```

Comment:
can we just say `keeper`? we always do that for the keeper of the module we are in.